packages/libs/eda/src/lib/core/components/visualizations/OutputEntityTitle.tsx

```diff
@@ -26,10 +26,11 @@ export function OutputEntityTitle({ entity, outputSize, subtitle }: Props) {
           : 'No entity selected'}
       </span>
       {subtitle && (
-        <div className={cxSubtitle()} style={{ color: gray[700] }}>
-          <br />
-          {subtitle}
-        </div>
+        <div
```

Comment:
Then I think you can just revert this change and go back to `{subtitle}` and get out of the Danger Zone...